vmdb/lib/miq_automation_engine/engine/miq_ae_method.rb

```diff
@@ -131,6 +131,7 @@ class AutomateMethodException < StandardError
   Socket.do_not_reverse_lookup = true  # turn off reverse DNS resolution
 
   require 'drb'
+  require 'yaml'
```

Comment:
yeah, i'm just wondering if yaml is the only direct dependency that we relied on activesupport to require.